src/test/java/com/bloxbean/cardano/client/backend/api/helper/UtxoTransactionBuilderTest.java

```diff
@@ -346,7 +347,7 @@ public void testBuildTransactionWithMultiAssetAndNotSufficientLovelaceWillReturn
                 .sender(sender)
                 .fee(BigInteger.valueOf(12000))
                 .unit(LOVELACE)
-                .amount(BigInteger.valueOf(1700000))
+                .amount(BigInteger.valueOf(3000000))
```

Comment:
@satran004 as one less utxo was created by the transaction builder, there were about 1.3 additional $ada required to use an additional utxo. I've just increased the amount to spend of a comparable value, this had the desired effect of consuming 3 utxos, as per the original intent of the test.